models/apartment.rb

```diff
@@ -0,0 +1,94 @@
+require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
+apartments = data[:apartments]
+tenants = data[:tenants]
+
+ruby_apartments = []
+ruby_tenants = []
+
+
+class Apartment
+
+  def initialize(id, address, monthly_rent, square_feet)
+    @id = id
+    @address = address
+    @monthly_rent = monthly_rent
+    @square_feet = square_feet
+    pick_option()
+    # puts "Apartment #{id} has been added"
+  end
+
+  def id
+    return @id
+  end
+
+  def address
+    return @address
+  end
+
+  def monthly_rent
+    return @monthly_rent
+  end
+
+  def square_feet
+    return @montly_rent
+  end
+
+  def pick_option
```

Comment:
This sort of code is more about the user interface, so I'd want to have this in a different file, like an `app.rb`.